projects/aca-content/src/lib/components/search/search-input/search-input.component.spec.ts

```diff
@@ -194,3 +207,79 @@ describe('SearchInputComponent', () => {
     });
   });
 });
+
+describe('navigateToSearch()', () => {
```

Comment:
we have 1 root describe per test file, this should be a child describe, and without a duplicate testbed setup